src/GitHub.App/ViewModels/PullRequestListViewModel.cs

```diff
@@ -68,6 +68,10 @@ public class PullRequestListViewModel : BaseViewModel, IPullRequestListViewModel
                 OrderedComparer<IAccount>.OrderByDescending(x => x.Login).Compare);
             trackingAssignees = new TrackingCollection<IAccount>(Observable.Empty<IAccount>(),
                 OrderedComparer<IAccount>.OrderByDescending(x => x.Login).Compare);
+
+            trackingAuthors.NewerComparer = OrderedComparer<IAccount>.OrderByDescending(x => x.CreatedAt).Compare;
+            trackingAssignees.NewerComparer = OrderedComparer<IAccount>.OrderByDescending(x => x.CreatedAt).Compare;
+
```

Comment:
Adding this `NewerComparer` will prevent `CopyFrom()` being called multiple times if a user is the author/assignee of multiple pull requests in the set.